modules/common/istio/istio-values.tpl

```diff
@@ -35,6 +37,7 @@ global:
       limits:
         cpu: 40m
         memory: 64Mi
+    protocolDetectionTimeout: 100ms 
```

Comment:
weren't there more updates in the values file?

Reply:
Yes but we don't have the parent stanzas in our tpl file at all so I opted to keep in line with what others before me did.